src/Aquifer.Public.API/Endpoints/Bibles/Texts/Get/Endpoint.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.Texts.Get;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, Response>
+{
+    private readonly AquiferDbContext _dbContext = dbContext;
+    public override void Configure()
+    {
+        Get("/bibles/{BibleId}/texts");
+        Description(d => d.ProducesProblemFE());
```

Comment:
I didn't add caching here even though I saw it elsewhere because the query string parameters substantially impact the response.  Is that handled by the default caching?

Reply:
Yeah the caching is setup to differ based on any query param. Probably need to double check that is working in the Public API though. I know it works in the internal one.

Reply:
I don't think there's any value in caching these. The use case isn't likely going to be someone hitting the same ones over and over again, or different actors hitting it close together.